ts/IconButton/index.tsx

```diff
@@ -1,50 +1,53 @@
-import {MDCIconButtonToggle} from '@material/icon-button';
-import {bind} from 'bind-decorator';
 import {h} from 'preact';
 import MaterialComponent from '../Base/MaterialComponent';
-import Icon from '../Icon';
+import {MDCIconButtonToggle} from '@material/icon-button';
 
 export interface IIconButtonProps {
-  'data-toggle-on-content'?: boolean;
-  'data-toggle-off-content'?: boolean;
-  'data-toggle-on-label'?: boolean;
-  'data-toggle-off-label'?: boolean;
   disabled?: boolean;
 }
 
 export interface IIconButtonState {}
 
-export class IconButton extends MaterialComponent<
-  IIconButtonProps,
-  IIconButtonState
-> {
-  public static readonly Icon = Icon;
+export interface IButtonProps {
+  on?: boolean;
+  element?: boolean;
+}
+
+export interface IButtonState {}
+
+class Button extends MaterialComponent<IButtonProps, IButtonState> {
```

Comment:
Shouldn't that be called `Icon`? And also export this

Reply:
actually yes